infra/modules/network/resources/variables.tf

```diff
@@ -26,3 +26,9 @@ variable "name" {
   type        = string
   description = "Name to give the VPC. Will be added to the VPC under the 'network_name' tag."
 }
+
+variable "enable_waf" {
+  type        = bool
+  description = "Whether to enable AWS Web Application Firewall (WAF) for ALBs in this network."
+  default     = true
+}
```

Comment:
Why is this here? Don't add this. Don't change the plan unless I told you to do so